### ISSUE TITLE
[Accessibility] BreadcrumbItem do not respond to Enter Press

### ISSUE THREAD

**Issue by SureneruS** [OfficeDev/office-ui-fabric-react#969]
### Bug Report

- __Package version(s)__: 1.2.2
- __Browser and OS versions__: Chrome, Win10

#### Describe the issue:

IBreadcrumbItem interface takes onClick handler. It should be called when Enter is pressed, similar to buttons.

#### Actual behavior:

No action when Enter is pressed.

#### Expected behavior:

Enter should trigger onClick event,

### If applicable, please provide a codepen repro:

Keyboard interactions are not working in codepen
